src/ui-test/test-data/community/spring-url-2.24.2.json

```diff
@@ -0,0 +1,5 @@
+{
+    "framework": "spring",
+    "type": "url",
+    "camelVersion": "2.24.2"
```

Comment:
maybe we should use the latest 2.x version? what do you think?

Reply:
Agree, changed version to 2.25.0.

Reply:
reflect this version change also in all readme files please